### PR TITLE
Automated Migration Testing (HAPI-FHIR) V7_4_0

### PR DESCRIPTION
**Pull request by volodymyr-korzh**
Migration test scripts added/changed in scope of this PR:

V7_2_0:
- [x] Updated: `TRM_CONCEPT` table. Added: `PARENT_PIDS_VC` column
- [x] Updated: `TRM_VALUESET_CONCEPT` table. Added: `SOURCE_DIRECT_PARENT_PIDS_VC` column
- [x] Updated: `TRM_CONCEPT_PROPERTY` table. Added: `PROP_VAL_BIN` column
- [x] Updated: `HFJ_BINARY_STORAGE_BLOB` table. Added: `STORAGE_CONTENT_BIN` column

V7_4_0:
- [x] Updated: `HFJ_RES_SEARCH_URL` table. Added: `PARTITION_ID`, `PARTITION_DATE` columns
- [x] Updated: `HFJ_IDX_CMP_STRING_UNIQ` table. Added: `HASH_COMPLETE`, `HASH_COMPLETE_2` columns
- [x] Updated: `TRM_CONCEPT_DESIG` table. Added: `VAL_VC` column

### CODE REVIEW

File: hapi-fhir-jpaserver-test-utilities/src/test/java/ca/uhn/fhir/jpa/embedded/HapiSchemaMigrationTest.java [hunk 1]
```diff
@@ -171,7 +171,7 @@ private void verifyHfjResSearchUrlMigration(JpaEmbeddedDatabase theDatabase, Dri
 
 		final Object allCountValue = allCount.get(0).values().iterator().next();
 		if (allCountValue instanceof Number allCountNumber) {
-			assertThat(allCountNumber.intValue()).isEqualTo(1);
+			assertThat(allCountNumber.intValue()).isEqualTo(2);
```
Comment (nathandoef) [inline hapi-fhir-jpaserver-test-utilities/src/test/java/ca/uhn/fhir/jpa/embedded/HapiSchemaMigrationTest.java:174]: Question: Why did you have to change these assertions?

**Review by nathandoef — APPROVED** [review]
Approved pending one question

File: hapi-fhir-jpaserver-test-utilities/src/test/java/ca/uhn/fhir/jpa/embedded/HapiSchemaMigrationTest.java (hunk 1 above)
Comment (volodymyr-korzh) [reply hapi-fhir-jpaserver-test-utilities/src/test/java/ca/uhn/fhir/jpa/embedded/HapiSchemaMigrationTest.java:174]: This validation checks the number of entries in `HFJ_RES_SEARCH_URL` table. As I added one more row to `HFJ_RES_SEARCH_URL` table in scope of `V7_4_0` migration changes - I had to adjust this check.

Comment (nathandoef) [reply hapi-fhir-jpaserver-test-utilities/src/test/java/ca/uhn/fhir/jpa/embedded/HapiSchemaMigrationTest.java:174]: Ahh ok, makes sense

**Pull request merged**